src/test/Checkpoints_tests.cpp

```diff
@@ -22,7 +22,7 @@ BOOST_FIXTURE_TEST_SUITE(Checkpoints_tests, BasicTestingSetup)
 /*
 BOOST_AUTO_TEST_CASE(sanity)
 {
-    const Checkpoints::CCheckpointData& checkpoints = Params(CBaseChainParams::MAIN).Checkpoints();
+    const CCheckpointData& checkpoints = Params(CBaseChainParams::MAIN).Checkpoints();
```

Comment:
Nits:

1) Instead of updating this file, just remove it. Checkpoints "have been removed for now" for 2 years.
2) Making checkpoints a reference is safe (the lifetime of the temporary is extended) but relying on that is weird and making this a value would have zero cost and make the code clearer.

Reply:
This test is removed entirely upstream in https://github.com/bitcoin/bitcoin/pull/9053, so I'm +0 on whether we fix this test to run as-is (it was commented out before launch, when we had no checkpoints, but could trivially be re-enabled now), or remove it.